README.md

```diff
@@ -54,7 +54,7 @@ To persist `libphonenumber\PhoneNumber` objects, add the `Misd\PhoneNumberBundle
     doctrine:
         dbal:
             types:
-                phone_number: Misd\PhoneNumberBundle\Doctrine\DBAL\Types\PhoneNumberType
+                phonenumber: Misd\PhoneNumberBundle\Doctrine\DBAL\Types\PhoneNumberType
```

Comment:
So ? With types: phone_number:  I have an error when I request a doctrine:schema:update , 
with phonenumber:  I don't .... and the application works, I go the views rendered with
`{{ phone_number_format(migrant.phonenumber, 'INTERNATIONAL') }}`
, There is still a problem, the   doctrine:schema:update works. If I look into the MySQL database, I find a field declaration
`phonenumber	varchar(35)	utf8_unicode_ci `
But the 
`$ app/console doctrine:schema:update --dump-sql`  shows 
`ALTER TABLE migrant CHANGE phonenumber phonenumber VARCHAR(35) DEFAULT NULL;`
@:/var/www/mdm$ app/console doctrine:schema:update --force
```
Updating database schema...
Database schema updated successfully! "2" queries were executed
```
even after an update ... 
luc@luc-ThinkPad-X201:/var/www/mdm$ app/console doctrine:schema:update --dump-sql
ALTER TABLE migrant CHANGE phonenumber phonenumber VARCHAR(35) DEFAULT NULL;

So ?